crates/bevy_transform/src/systems.rs

```diff
@@ -1,10 +1,5 @@
 use crate::components::{GlobalTransform, Transform};
-use bevy_ecs::{
-    entity::Entity,
-    prelude::Changed,
-    query::{With, Without},
-    system::Query,
-};
+use bevy_ecs::prelude::*;
```

Comment:
Why this change?
The style guide mention preferring granular imports
https://github.com/bevyengine/bevy/blob/7d21ca37446b04ae68f7ee092f545fb6dc2a40fb/.github/contributing/engine_style_guide.md?plain=1#L9

Reply:
I wasn't aware of that guideline. I made this change here because I think I needed something else in the import, and importing the prelude is significantly less prone to merge conflicts overall.

As a note, that style guide seems to only care about cases where you have a dependency on the root `bevy` facade, not when depending directly on the sub crates.

I'm not sure that I really like that guideline - merge conflicts in the imports create a ton of busywork, since the tooling requires a lot of manual handholding.

I can change it back though.

Reply:
I don't like that rule for `bevy_ecs::prelude` - granular imports from other crates are more reasonable (`bevy_app` is marginal)

`bevy_ecs`'s types are the standard vocabulary of bevy, and there should be very few source files which don't use any of them (and where the set used is static). That is, I think glob importing from `bevy_ecs::prelude` should be encouraged. 

I think similarly for `bevy_app::prelude` could be argued, since `bevy_app` and `bevy_ecs` should be merged anyway, although the set used from `bevy_app` is more likely to be static.

Reply:
> As a note, that style guide seems to only care about cases where you have a dependency on the root bevy facade, not when depending directly on the sub crates.

Yeah, the example in the style guide is wrong, you never import bevy when working in bevy

> I don't like that rule for bevy_ecs::prelude - granular imports from other crates are more reasonable (bevy_app is marginal)
Maybe, but I don't think this PR is the place to change that and I would rather have consistency until we decide to change the rule

Reply:
#4644 discusses this change.